config/serverless.es.yml

```diff
@@ -65,7 +65,7 @@ data_visualizer.resultLinks.fileBeat.enabled: false
 xpack.searchPlayground.ui.enabled: true
 
 # Search InferenceEndpoints
-xpack.searchInferenceEndpoints.ui.enabled: false
+xpack.searchInferenceEndpoints.ui.enabled: true
```

Comment:
Based on [the recommendation #5](https://docs.google.com/document/d/1zFYwPLzg4deeVlE8pAaWYvl5_WZZu_VmTFV7K9H90j8/edit), we may need to hide the inference endpoints page in the servereless environment.

Reply:
yes, we will not be merging this PR as the requirements changed

Reply:
@Samiul-TheSoccerFan we can merge this PR, we just need to not enable the FF. instead run the new FTRs in the feature flag config and only enable it there.